app/frontend/components/pages/login/ledgerTransportationSelect.tsx

```diff
@@ -0,0 +1,36 @@
+import {h} from 'preact'
+import SearchableSelect from '../../common/searchableSelect'
+import {LedgerTransportType} from '../../../../frontend/types'
+import styles from './ledgerTransportationSelect.module.scss'
+
+interface Props {
+  onSelect: (ledgerTransportType: LedgerTransportType) => void
+}
+
+const LedgerTransportationSelect = ({onSelect}: Props) => {
+  const dropdownAssetItems = [
+    LedgerTransportType.DEFAULT,
+    LedgerTransportType.U2F,
+    LedgerTransportType.WEB_USB,
+    LedgerTransportType.WEB_HID,
+  ]
+
+  const displayItem = (item: LedgerTransportType) => <div>{item}</div>
+
+  return (
+    <SearchableSelect
```

Comment:
why a searchable select? I'd expect an ordinary one (or the name is misleading)

Reply:
Because we don't have "ordinary" one. We already have this one "searchable" which can be easily used without the search functionality.